sdk/storage/Azure.Storage.Queues/api/Azure.Storage.Queues.netstandard2.0.cs

```diff
@@ -368,6 +377,7 @@ public partial class SpecializedQueueClientOptions : Azure.Storage.Queues.QueueC
     }
     public static partial class SpecializedQueueExtensions
     {
+        public static Azure.Storage.Queues.QueueServiceClient GetParentQueueServiceClient(this Azure.Storage.Queues.QueueClient client) { throw null; }
```

Comment:
This is similar to https://github.com/Azure/azure-sdk-for-net/pull/16437 .
While working on downstream stuff I figured that this will be extremely useful if someone want's to jump from `InvalidMessageEventArgs.QueueClient` to some other queue (i.e. poison queue).

Reply:
This is nice.  Should we add an extension method in WebJobs to get the poison queue for a QueueClient?

Reply:
Good idea. In webjobs PR though.